rotkehlchen/tests/api/test_custom_ethereum_tokens.py

```diff
@@ -386,7 +386,7 @@ def test_deleting_custom_tokens(rotkehlchen_api_server):
     underlying3_id = ETHEREUM_DIRECTIVE + underlying_address3
     cursor = GlobalDBHandler()._conn.cursor()
     initial_underlying_num = cursor.execute('SELECT COUNT(*) from underlying_tokens_list').fetchone()[0]  # noqa: E501
-    assert initial_underlying_num == 3, 'check underlying tokens mapping start number is expected'
+
```

Comment:
Why are you doing this? Afaics this assertion should still hold.

Reply:
This one doesn't is true but I don't see how to check it with the db initialized as it is with underlying tokens